go/billing/api.py

```diff
@@ -478,12 +482,25 @@ def create_transaction(self, account_number, message_id, tag_pool_name,
         defer.returnValue(result)
 
 
+class HealthResource(Resource):
+    isLeaf = True
+
+    def __init__(self, health_check_func):
+        Resource.__init__(self)
+        self._health_check_func = health_check_func
+
+    def render_GET(self, request):
+        self._health_check_func(request)
+        return NOT_DONE_YET
+
+
 class Root(BaseResource):
     """The root resource"""
 
     def __init__(self, connection_pool):
         BaseResource.__init__(self, connection_pool)
         self.putChild('transactions', TransactionResource(connection_pool))
+        self.putChild('ping', HealthResource(self.health_check))
```

Comment:
I have no personal objections to ``ping``, just mentioning that we've usually used ``health`` as the URL path.